spec/components/stylesheet/manager_spec.rb

```diff
@@ -135,6 +135,67 @@ def manager(theme_id = nil)
       )
     end
 
+    context "stylesheet order " do
+      let(:z_child_theme) { Fabricate(:theme, component: true, name: "ze component").tap { |z|
+        z.set_field(target: :desktop, name: "scss", value: ".child_desktop{.scss{color: red;}}")
+        z.save!
+      }}
```

Comment:
```suggestion
      let(:z_child_theme) do 
        Fabricate(:theme, component: true, name: "ze component").tap do |z|
          z.set_field(target: :desktop, name: "scss", value: ".child_desktop{.scss{color: red;}}")
          z.save!
        end
      end
```

We usually prefer `do..end` for multi-line blocks